app/views/pivotal/index.html.erb

```diff
@@ -48,9 +48,7 @@
 
 <div class="row">
   <div class="col-xs-12 max-width-text">
-    <% if @stories.count > 900 %>
-        <b>Displaying <%= @stories.count %> of 1,000 available Feature Requests.</b>
-    <% end %>
+      <b>Displaying <%= @stories.count %> of 1,000 available Feature Requests. Please <%= link_to 'let the admins know', '/observer/ask_webmaster_question', title: "Contact Us" %> that we're close to the limit.</b>
```

Comment:
Wait... don't we want to keep the if statement?  I just want to someone to warn me when we get close to the limit, in case I don't look at the page myself regularly.